docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_RNNT.md

```diff
@@ -0,0 +1,102 @@
+# Convert ONNX* RNNT to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_RNNT}
+
+This instruction covers conversion of RNN-t model from [MLCommons](https://github.com/mlcommons) repository. Before
+converting into IR PyTorch model need to be exported into ONNX. In order to that follow the steps below.
+
+**Step 1**. Clone RNN-t PyTorch implementation from MLCommons repository. In order to pull only RNN-t model 
+without full repository make a shallow clone. If you alreade have a full repository skip this and go to **Step 2**:
+```bash
+git clone -n https://github.com/mlcommons/inference rnnt_for_openvino --depth 1
+cd rnnt_for_openvino
+git checkout HEAD speech_recognition/rnnt 
+```
+
+**Step 2**. If you already have a full clone of MLCommons inference repository create a folder for 
+pre-trained PyTorch model and where conversion into IR will take place. Skip this step if you did a shallow clone.
```

Comment:
a folder for pre-trained PyTorch model and where conversion into IR will take place ---> a folder for
pretrained PyTorch model, where conversion into IR will take place.
if you did a shallow clone ---> if you have a shallow clone